README/MATH/Prediction_Market_Math.tex

```diff
@@ -0,0 +1,1379 @@
+{
+    \draw (\x,-9) rectangle ++(1,1);
+    \filldraw[fill=blue!50] (\x,-9) rectangle ++(1,1);
+}
+
+
+
+% Label for Course Payout of Bet 3
+\node[above] at (-4,-8) {Course Payout for Bet 3 (\( C_1 = 1.25 \) units)};
+
+
+% Bet 4 YES (Red Squares)
+\foreach \x in {5,...,14}
+{
+    \draw[fill=pink!50] (\x,-13) rectangle ++(1,1);
+}
+
+% Label for Bet 4
+\node[above] at (11.2,-12) {Bet 4: NO (\( b_1 = -10 \) units at \( p_1 = 0.625 \))};
+
+% Course Payout for Bet 4
+\def\xPartial{-3}  % Position of the partial square
+\draw (\xPartial,-13) rectangle ++(1,1);
+
+% Label for Course Payout of Bet 4
+\node[above] at (-4,-12) {Course Payout for Bet 4 (\( C_1 = 0.0 \) units)};
+
+
+
+\end{tikzpicture}
+\end{center}
+
+\section*{Normalization Factor Calculation for Fourth Transaction}
+
+\subsection*{Step Three: Calculate Normalization Factor}
+
+Given:
+\begin{itemize}
+    \item Total YES Shares: \( S_{\text{YES}} = 25 \)
+    \item Total NO Shares: \( S_{\text{NO}} = 15 \)
+    \item Course Payouts:
+    \begin{itemize}
+        \item \( C_{\text{YES,1}} = 2.5 \) units
+        \item \( C_{\text{NO,1}} = 9.16 \) units
+        \item \( C_{\text{YES,2}} = 1.25 \) units
+        \item \( C_{\text{NO,2}} = 0.0 \) units
+        \item Total payout sums: \( C_{\text{YES\_SUM}} = 3.75 \), \( C_{\text{NO\_SUM}} = 9.16 \)
+    \end{itemize}
+\end{itemize}
+
+\subsubsection*{Calculate YES Normalization Factor}
+
+\[
+F_{\text{YES}} =
+\begin{cases}
+      \frac{S_{\text{YES}}}{C_{\text{YES\_SUM}}} & \text{if } C_{\text{YES\_SUM}} > 0 \\
+      0 & \text{if } C_{\text{YES\_SUM}} = 0
+   \end{cases}
+\]
+
+Since \( C_{\text{YES\_SUM}} = 3.75 \), we have:
+
+\[
+F_{\text{YES}} = \frac{25}{3.75} = 6.67
+\]
+
+\subsubsection*{Calculate NO Normalization Factor}
+
+\[
+F_{\text{NO}} =
+\begin{cases}
+      \frac{S_{\text{NO}}}{C_{\text{NO\_SUM}}} & \text{if } C_{\text{NO\_SUM}} > 0 \\
+      0 & \text{if } C_{\text{NO\_SUM}} = 0
+   \end{cases}
+\]
+
+Since \( C_{\text{NO\_SUM}} = 9.16 \), we have:
+
+\[
+F_{\text{NO}} = \frac{15}{9.16} \approx 1.64
+\]
+
+Thus, the normalization factors are:
+
+\[
+F_{\text{YES}} \approx 6.67, \quad F_{\text{NO}} \approx 1.64
+\]
+
+
+\section*{Scaled Payout Calculation for Fourth Transaction}
+
+\begin{itemize}
+    \item \textbf{Fourth Transaction:}
+    \begin{itemize}
+        \item We can now see the effects of the course payout on scaling toward older transactions. While both Bet 2 and Bet 3 were of equal amounts, because Bet 2 was made at at a point when the YES outcome was less likely based upon the price, there is proportionally more weight given to this bet.
+        \item The fourth bet is a negative amount because it was a sale. Because the fourth bet occurs at the resolution price R, the Scaled Payout is zero. However in a future transaction, the Scaled Payout for this transaction could become negative, which would reduce the user's final payout, because they are no longer participating in the market at this amount.
```

Comment:
This third sentence is loooooong and should be split into 2

e.g. In a future transaction, the Scaled Payout could become negative. This would reduce the user's final payout because [and then the last clause needs to be made clearer]